test/TensorFlowNET.UnitTest/PythonTest.cs

```diff
@@ -196,17 +245,25 @@ public T evaluate<T>(Tensor tensor)
             //    return self._eval_helper(tensors)
             //  else:
             {
-                var sess = tf.Session();
```

Comment:
@Wanglongzhi2001 `ts.Session()` produces a new session where we need to get a cached one.